sample-mod/versions.properties

```diff
@@ -0,0 +1,18 @@
+## suppress inspection "SpellCheckingInspection" for whole file
+##
+## Dependencies and Plugin versions with their available updates
+## Generated by $ ./gradlew refreshVersions
+## Please, don't put extra comments in that file yet, keeping them is not supported yet.
+
+plugin.fabric-loom=0.5.12
+
+plugin.org.jetbrains.kotlin.jvm=version.kotlin
+
+version.io.github.prospector.modmenu..ModMenu=1.6.3-95
+
+version.kotlin=1.4.0
+
+version.net.fabricmc..fabric-loader=0.8.8+build.202
+##                      # available=0.9.1+build.205
+
+version.net.fabricmc.fabric-api..fabric-api=0.13.1+build.370-1.16
```

Comment:
and this

Reply:
originally i used 0.18.0+build.397-1.16 but that crashed when used with 1.16 as it required 1.16.2

Reply:
Yes, it would do. Just use 1.16.2 everywhere I see no reason to use an older version

Reply:
i tried t stupid thing of picking the version based on `substringAfterlast('-')` on that version
trying to make it automatically pick the correct one, but that went wrong

Reply:
removed that anyways